placeholder-sql-azure-pipelines.yml

```diff
@@ -0,0 +1,147 @@
+parameters:
+- name: OverrideBlockOnPossibleDataLoss
+  displayName: Add /p:BlockOnPossibleDataLoss=false argument to DACPAC deployment
+  type: boolean
+  default: false
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+variables:
+- name: SolutionBaseName
+  value: __SolutionBaseName__ #PLACEHOLDER
+- name: DACPACSolutionBaseName
+    value: __DACPACSolutionBaseName__ #PLACEHOLDER
+- name: BuildConfiguration
+  value: release
+- name: BuildPlatform
+  value: any cpu
+- group: RELEASE Management Resources
+- group: __ReleaseVariableGroupName__ #PLACEHOLDER
+- name: Deploy
+  value: $[or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), eq(variables['Build.Reason'], 'Manual'), eq(variables['Build.Reason'], 'PullRequest'))]
+
+
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.4.26
+    endpoint: SkillsFundingAgency
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.5.10
```

Comment:
can we have placeholder versions so people are encouraged to get the latest version?